CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+Contributing to Okta Open Source Repos
+
+## Style
+
+### Git Commit Messages
+
+We use a simplified form of [Atom](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#git-commit-messages) commit convention.
+
+  * Use the present tense ("Add feature" not "Added feature")
```

Comment:
Wouldn't it be "Adds feature" "Fixes bug X"? As in "This commit adds feature X."

Also, "Add feature" is an imperative (or rather, impossible to distinguish from the imperative form without more context."